contracts/src/chainlink/cairo/access/ownable.cairo

```diff
@@ -22,6 +23,9 @@ end
 func Ownable_only_owner{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}():
     let (owner) = Ownable_owner.read()
     let (caller) = get_caller_address()
+    with_attr error_message("Ownable: Caller is the zero address"):
+        assert_not_zero(caller)
+    end
```

Comment:
I wanted to keep it as similar as possible to the OpenZeppelin version: https://github.com/OpenZeppelin/cairo-contracts/blob/b324794e5337ce72d65266df660e4918fde60c70/src/openzeppelin/access/ownable/library.cairo#L44-L58

They deliberately split the two so that zero address has a more distinct error message: https://github.com/OpenZeppelin/cairo-contracts/pull/422